LCU.NET.Extras/Game/Actuator.Game.cs

```diff
@@ -203,29 +203,23 @@ public async Task UploadItemSet (Position pos, int championId)
 
 		public async Task UploadRunes (Position pos, int championId)
 		{
-			LogTo.Debug ($"Trying to upload rune page for champion {championId}");
+			if (!Config.Current.LoadRunesOnLock) {
+				return;
+			}
 
-			var page = RuneBook.Instance.Get (championId, pos, false);
+			LogTo.Debug ($"Trying to upload rune page for champion {championId}");
 
+			LogTo.Info ("Downloading from provider");
+			Provider provider = GetProvider ();
+			RunePage page = await Main.LoadPageFromProvider (provider, championId);
 			if (page == null) {
-				LogTo.Info ("Invalid current rune page");
-
-				if (Config.Current.LoadOnLock) {
-					LogTo.Info ("Downloading from provider");
-					Provider provider = GetProvider ();
-					page = await Main.LoadPageFromProvider (provider, championId);
-					if (page == null) {
-						var positionName = pos.ToString ().ToLower ();
-						LogTo.Debug ($"Could not retrieve a valid rune page for champion {championId} in {positionName}");
-						Main.ShowNotification ($"No runes for {positionName}", $"There's not enough info to set runes for {championId} in {positionName}", NotificationType.Error);
-						return;
-					}
-					LogTo.Debug ("Downloaded from provider");
-				} else {
-					Main.ShowNotification (string.Format ("Rune page for champion {0} not set", championId), null, NotificationType.Error);
-					return;
-				}
+
+				var positionName = pos.ToString ().ToLower ();
+				LogTo.Debug ($"Could not retrieve a valid rune page for {championId} in {positionName}");
+				Main.ShowNotification ($"No runes for {positionName}", $"There's not enough info to set runes for {championId} in {positionName}", NotificationType.Error);
```

Comment:
Main.ShowNotification ($"No runes for position {positionName}", $"There's not enough info to set runes for your champion in position {positionName}", NotificationType.Error);